README.md

```diff
@@ -95,12 +101,7 @@ iex> Catalog.refresh
 - [ ] Support all options to refresh and create views
 - [ ] Implement automatic view versioning for migration
 
-## Docs
-
-Documentation can be generated with [ExDoc](https://github.com/elixir-lang/ex_doc)
-and published on [HexDocs](https://hexdocs.pm). Once published, the docs can
-be found at [https://hexdocs.pm/ecto_vista](https://hexdocs.pm/ecto_vista).
-
 ## License
 
 The source code is under the Apache 2 License.
+Copyright (c) 2019- Nick Chernyshev.
```

Comment:
Let's use the full name for legal things 😄 
```suggestion
Copyright ©2019 Nikolai Chernyshev
```

Reply:
Definitely looks more professional and convincing. :smile: